lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,15 +1,22 @@
 import React, { Component } from 'react';
 import Card from './Card';
+import PropTypes from "prop-types";
 
 const Cards = props => {
   return (
     <div className="cards-container">
       {/* Using the cards prop, map over the list creating a 
-          new Card component for each passing the card as the only prop*/}
+          new Card component for each passing the card as the only prop*/
+          props.cards.map(card => {
+            return(<Card key={Date.now() * Math.random()} card = {card} />)
+          })}    
     </div>
   )
 }
 
 // Make sure you include prop types for all of your incoming props
+Cards.propTypes = {
+  cards: PropTypes.arrayOf(PropTypes.object)
+}
```

Comment:
you could use the shape propType here for the object to be even more explicit here